libraries/ms-drivers/inc/adt_7476a_fan_controller.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "adc.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "i2c.h"
+#include "interrupt.h"
+#include "soft_timer.h"
```

Comment:
These headers should be included in the .c file rather than the .h file (except the ones you need like gpio and i2c) also why do you need the adc header?